src/platform/utilities/error-boundary/index.jsx

```diff
@@ -0,0 +1,23 @@
+// libs
+import React from 'react';
+import * as Sentry from '@sentry/react';
+
+// default fallback component
+function FallbackComponent() {
+  return <div />;
+}
+
+function ErrorBoundary(props) {
+  const { children, fallback, showDialog = false } = props;
+
+  return (
+    <Sentry.ErrorBoundary
+      fallback={fallback || FallbackComponent}
+      showDialog={showDialog}
+    >
+      {children}
+    </Sentry.ErrorBoundary>
+  );

```

Comment:
So what exactly is this doing for us?

Reply:
Starting to leverage error boundaries in react. This is Sentry's wrapper around it. 
https://reactjs.org/docs/error-boundaries.html